lib/watir-webdriver/locators/element/selector_builder/css.rb

```diff
@@ -0,0 +1,63 @@
+module Watir
+  class Element
+    class SelectorBuilder
+      class CSS
```

Comment:
I think we should get rid of the prefer_css toggle entirely. There is no performance improvement and even with this refactor I think it unnecessarily complicates things.